src/System.Data.SqlClient/tests/ManualTests/SQL/ConnectionPoolTest/ConnectionPoolTest.cs

```diff
@@ -54,7 +54,7 @@ private static void RunDataTestForSingleConnString(string tcpConnectionString, s
         {
             BasicConnectionPoolingTest(tcpConnectionString);
             ClearAllPoolsTest(tcpConnectionString);
-#if MANAGED_SNI
+#if MANAGED_SNI && DEBUG
```

Comment:
Nit: 
Instead of wrapping the method call in the DEBUG flag, would it be better to wrap the function body in DEBUG ?